css/mainbody.css

```diff
@@ -35,19 +34,41 @@ body {
 	border-radius: 25px;
 }
 
-#homepageimg img {
-	width: 250px;
-	height: 250px;
-	margin: auto;
-	margin-left: 25px;
-	margin-top: 25px;
-	-webkit-border-radius: 35px;
-	-moz-border-radius: 35px;
-	-m-border-radius: 35px;
-	-o-border-radius: 35px;
-	border-radius: 35px;
+.blogsnippet {
+	border: 3px solid black;
+	height: 80px;
+	width: 480px;
+	margin: 10px 0 10px 10px;
+	line-height: 50%;
+	overflow: hidden;
+	font-size: 13px;
+}
+
+.blogsnippet h5 {
+	margin: 15px;
+	font-size: 20px;
 }
 
+.blogsnippet h6{
+	margin: 0 0 0 15px;
+	font-size: 15px;
+}
+
+.blogsnippet p {
```

Comment:
This and .blogsnippet h6 are exactly the same - either join specify both classes, or use one class for both.